blueocean-github-pipeline/pom.xml

```diff
@@ -21,21 +21,16 @@
     </dependency>
 
     <dependency>
-      <groupId>org.kohsuke</groupId>
-      <artifactId>github-api</artifactId>
-      <version>1.82</version>
-    </dependency>
-
-    <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>github-organization-folder</artifactId>
-      <version>1.5</version>
+      <version>1.6</version>
+      <optional>true</optional>
```

Comment:
This will still pull in the dep in new installs, right? Do we plan to delete this dep at some point in the future?